### PR TITLE
Visual Editor - Add new fields for position-type DOM mutations

### PR DESCRIPTION
**Pull request by bttf**
This adds two new fields to our VisualChangeset model to accommodate position-style DOM mutations

- `parentSelector`
- `insertBeforeSelector`

### CODE REVIEW

**Review by bryce-fitzsimons — DISMISSED** [review]

File: packages/back-end/types/visual-changeset.d.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ interface DOMMutation {
   action: "append" | "set" | "remove";
   attribute: string;
   value: string;
+  parentSelector?: string;
```
Comment (jdorn) [inline packages/back-end/types/visual-changeset.d.ts:6]: Should `value` be optional here too since it's not used all the time?

**Comment by jdorn** [comment]
Do we also need to update the openapi schema to support these new fields?

File: packages/back-end/types/visual-changeset.d.ts (hunk 1 above)
Comment (bttf) [reply packages/back-end/types/visual-changeset.d.ts:6]: Updated

**Comment by bttf** [comment]
> Do we also need to update the openapi schema to support these new fields?

Added

**Review by jdorn — APPROVED** [review]

**Pull request merged**